BZ:2049053: fix crio annotation description and examples (#41282)
openshift/openshift-docs: modules/cnf-managing-device-interrupt-processing-for-guaranteed-pod-isolated-cpus.adoc

To achieve low latency for workloads, some (but not all) pods require the CPUs they are running on to not process device interrupts. A pod annotation, `irq-load-balancing.crio.io`, is used to define whether device interrupts are processed or not. When configured, CRI-O disables device interrupts only as long as the pod is running.

[id="disabling-cpu-cfs-quota_{context}"]
== Disabling CPU CFS quota

> To reduce CPU throttling for individual guaranteed pods create a performance profile with the annotation `cpu-quota.crio.io: "disable"`, this disables CPU completely fair scheduler (CFS) quota at the pod run time. An example pod specification snippet that illustrates this is below:

Reviewer: ```suggestion
To reduce CPU throttling for individual guaranteed pods, create a performance profile with the annotation `cpu-quota.crio.io: "disable"`. This annotation disables the CPU completely fair scheduler (CFS) quota at the pod run time. The following pod specification contains this annotation:
```

Reviewer: > ! below adjective, adverb, preposition
Do not use to indicate a relative location in a document, as in "the information below".

[id="disabling_interrupt_processing_for_individual_pods_{context}"]
== Disabling interrupt processing for individual pods

> To disable interrupt processing for individual pods, ensure that `globallyDisableIrqLoadBalancing` is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`. The following pod specification contains this annotation:

Reviewer: ```suggestion
To disable interrupt processing for individual pods, ensure that the `globallyDisableIrqLoadBalancing` annotation is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`. The following pod specification contains this annotation:
```

> To disable interrupt processing for individual pods, ensure that `globallyDisableIrqLoadBalancing` is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`. An example pod specification snippet that illustrates this is below:

Reviewer: ```suggestion
To disable interrupt processing for individual pods, ensure that the`globallyDisableIrqLoadBalancing` annotation is set to `false` in the performance profile. Then, in the pod specification, set the `irq-load-balancing.crio.io` pod annotation to `disable`. The following pod specification contains this annotation:
```

I'm not sure what the difference between the "performance profile" and the "pod specification" are in this example. Should this YAML snippet also contain `globallyDisableIrqLoadBalancing`, or does it go somewhere else?

Reviewer: > Generally, specify the keyword type after the keyword.

Author: I had that same question, then I looked at the previously defined  `PerformanceProfile` so I felt it was OK not to repeat/add this here as the guidance states create a performance profile where `globallyDisableIrqLoadBalancing` is set to false

Reviewer: I'm more curious exactly where it should go in the performance profile.

Author: It would be like previous example, 
```
apiVersion: performance.openshift.io/v2
kind: PerformanceProfile
metadata:
  name: manual
spec:
  globallyDisableIrqLoadBalancing: false
``` 

We have a pod spec and performance profile spec and from what I can see you reference the performance profile spec when you are setting up the pod spec. 
```
spec:
    runtimeClassName: performance-<profile_name>
```
